frontend/src/pages/projects/AddProject.tsx

```diff
@@ -67,12 +67,13 @@ export default function AddProject() {
                            onChange={handleChange}
                            placeholder="project name"
                     />
+                    <p></p>
```

Comment:
Empty HTML-Tags should be deleted

Reply:
Thanks. Changed Css Margin :) and deleted <p> tag